src/Databricks/Az.Databricks.format.ps1xml

```diff
@@ -8,6 +8,9 @@
       </ViewSelectedBy>
       <TableControl>
         <TableHeaders>
+          <TableColumnHeader>
+            <Label>PeeringName</Label>
```

Comment:
Vnet peerings commands are removed from generating, but the models are still generated. That's why there are so many files in this PR